### ISSUE TITLE
Feature request: resolve comments

### ISSUE THREAD

**Issue by Johennes** [refined-bitbucket/refined-bitbucket#408]
Both GitHub and GitLab have functionality to mark comments as resolved. BitBucket is lacking this feature, however, and when you go into the third or fourth review round on a PR, the comments start cluttering up the diff and it gets hard to keep track of what has already been addressed and what not.

It'd be great if Refined BitBucket would add an option to mark a single comment as resolved. In order to resolve it for everyone viewing the PR, the extension could add a special trigger comment (e.g. `[resolved]`) and hide / collapse all threads that end with such a comment.

**Comment by reyronald** [comment]
Hey @Johennes ,

I had a call with the Bitbucket team a few weeks ago and they told me this was on their backlog. Unsure about the priority or the release date..

I'll wait a bit to see if it lands soon before implementing it here.

**Comment by Johennes** [comment]
Yeah, I had actually checked their Jira yesterday before reporting this issue. I only found tickets like https://jira.atlassian.com/browse/BCLOUD-6535, reported a long while ago, so it didn't seem to be on any near term roadmap. 😔
